pkg/scheduler/framework/types.go

```diff
@@ -49,6 +49,11 @@ const (
 	Add ActionType = 1 << iota
 	Delete
 
+	// UpdateOtherPod if only applicable for UnscheduledPod events.
+	// If you plugin uses UpdateOtherPod,
+	// it indicates that the rejection from it could be resolved by updating another unscheduled Pod.
+	UpdateOtherPod
```

Comment:
This looks weird.

Instead of this, maybe we can create in `PodItself` in `GVK`:
- `AssignedPod`: events to any assigned Pods.
- `PodItself`: events to the unscheduled Pod itself.
- `UnscheduledPod`: events to unscheduled Pods except the unscheduled Pod itself.

Reply:
Yes, I agree this looks weird🤦.
`PodItself` is a good suggestion.